src/data/docs/documentation.md

```diff
@@ -5,257 +5,232 @@ description: How to write Unison documentation and associate it with your code
 
+Let's look at the following docs example to see more of the documentation type's powers: 
 ```unison
-myFunction.doc =
-  [: @myFunction multiplies its argument by two. :]
-```
+{{
+  PartOfSpeech describes the nine ways a word can function in a phrase. Defined as: 
 
-The hyperlink is based on a hash of `myFunction` rather than its name, so if it's ever renamed or moved, the link will still work.
+  @source{type PartOfSpeech}
 
-### Insert the source code of another definition
+  {{furtherExposition}}
 
-You can include the full source code of any definition in your codebase using `@[source]` followed by the name of the definition. For example, to include the source code of `myFunction` in its own documentation:
+  {{examples}}
+}}
+unique type PartOfSpeech Text = 
+  Noun Text | 
+  Verb Text | 
+  Adjective Text | 
+  Adverb Text | 
+  Pronoun Text | 
+  Preposition Text | 
+  Conjunction Text | 
+  Determiner Text | 
+  Interjection Text 
 
-```unison
-[:
-  Multiplies its argument by two.
-  Here is its source code:
-
-  @[source] myFunction
-:]
-```
-
-When Unison displays this `Doc`, it will be expanded to:
-
-```
-Multiplies its argument by two. Here is its source code:
+examples = {{
+  Examples:
+    * @eval{Interjection "Zoinks!"} 
+    * This is going @eval{Adverb "swimmingly"} 
+    * These are @eval{Adjective "lovely"}
+}}
+ 
+furtherExposition = {{
+  There are *many* different interpretations of the possible parts of speech. [Learn more](https://en.wikipedia.org/wiki/Part_of_speech)}}
+}}
 
-myFunction x = x * 2
 ```
+We'll break this down further. 
 
-Note that the source code is not copied into the document, it's just linked to it. Only on display of the document do we see what the link expands to. So if the source code for `myFunction` ever changes, your documentation changes along with it. This kind of linking where the link is expanded on display is sometimes called *[transclusion](https://en.wikipedia.org/wiki/Transclusion).*
+### Modular Docs
+Notice how documentation can be composed of much smaller blocks which are assembled together in the final `PartOfSpeech.doc` term. It's common to build longer documents by including subdocuments via the `{{ subdoc }}` syntax. 
 
-### Insert the type signature of a term in the codebase
+### Style 
+The Unison doc format supports basic text formatting and styling.  Text can be \__bold__, \*italicized*, \~~strikethrough~~, or \`monospaced` and paragraphs are separated by one or more blank lines.
 
-You can include the type signature of any function or value in the codebase, with `@[signature]` followed by the name of the function or value. For example, to include the type of `myFunction` in its own documentation:
+### Lists 
+Bullet pointed lists and numbered lists are also supported in the documentation format. `*`, `+`, and `-` are all valid bullet points, though the variation will be normalized away by an eventual pretty-printer.  Numbered lists take the form: 
```

Comment:
Transition to numbered lists from bulleted lists is too abrupt here. First show bulleted lists, and then numbered lists, as separate examples.